### PR TITLE
Add default Brave shields

### PR DESCRIPTION
**Pull request by bbondy**
Requires: https://github.com/brave/brave-browser/pull/1100
Fix brave/brave-browser#1288

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Needed or QA/No-QA-Needed) to include the closed issue in milestone 

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

### CODE REVIEW

File: browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <template is="dom-repeat" items="[[fingerprintingControlTypes_]]">
+          <option value="[[item.value]]"
+              selected="[[controlEqual(item.value, fingerprintingControlType_)]]">
+            [[item.name]]
+          </option>
+        </template>
+      </select>
+    </div>
+    <settings-toggle-button id="httpsEverywhereControlType"
+        pref="{{prefs.brave.https_everywhere_default}}"
```
Comment (petemill) [inline browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html]: Correct me if I'm wrong, but wouldn't this key need to be added to the whitelist of pref keys that appear in the `settingsPrivate` JS api? A Brave override for this doesn't yet appear to exist so I added one in #454 here: https://github.com/brave/brave-core/pull/454/files#diff-3971ac612305df099f1e78861ce8b884R16

Just wondering if there's something I'm missing and this works here without the whitelist addition @bbondy...

Comment (bbondy) [reply browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html]: Possibly, I did have a setting not found error but I haven't looked into why yet.

Comment (petemill) [reply browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html]: I had the same error until I implemented the whitelist

Comment (bsclifton) [reply browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html]: @petemill does that whitelist get propagated to `CreateIncognitoPrefService`? If so, those entries will be persisted to disk (using the `JsonPrefStore`)

**Comment by bbondy** [comment]
Good to go

File: browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html (hunk 1 above)
Comment (bbondy) [reply browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html]: worked like a charm, thx @petemill

**Comment by bbondy** [comment]
master 245a4949098aa97de91917502c27ff3e7fff704d
0.55.x a6d9cee64ef29a8027b887b69612c12c142fe010

**Pull request merged**